Compiler warning about alignment

Building on macOS, I am getting the following warnings:

```
[46/129] Building CXX object tomviz/CMakeFiles/tomvizlib.dir/EmdFormat.cxx.o
/Users/cory.quammen/src/tomviz/tomviz/EmdFormat.cxx:270:38: warning: cast from 'float *' to 'VTK_TT *' (aka 'double *') increases required alignment from 4 to 8 [-Wcast-align]
                         writeVolume((VTK_TT*)(dataPtr), groupId, name.c_str(),
                                     ^~~~~~~~~~~~~~~~~~
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:745:44: note: expanded from macro 'vtkTemplateMacro'
  vtkTemplateMacroCase(VTK_DOUBLE, double, call);                           \
                                           ^
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:743:38: note: expanded from macro 'vtkTemplateMacroCase'
  case typeN: { typedef type VTK_TT; call; }; break
                                     ^
/Users/cory.quammen/src/tomviz/tomviz/EmdFormat.cxx:270:38: warning: cast from 'float *' to 'VTK_TT *' (aka 'long long *') increases required alignment from 4 to 8 [-Wcast-align]
                         writeVolume((VTK_TT*)(dataPtr), groupId, name.c_str(),
                                     ^~~~~~~~~~~~~~~~~~
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:747:50: note: expanded from macro 'vtkTemplateMacro'
  vtkTemplateMacroCase(VTK_LONG_LONG, long long, call);                     \
                                                 ^
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:743:38: note: expanded from macro 'vtkTemplateMacroCase'
  case typeN: { typedef type VTK_TT; call; }; break
                                     ^
/Users/cory.quammen/src/tomviz/tomviz/EmdFormat.cxx:270:38: warning: cast from 'float *' to 'VTK_TT *' (aka 'unsigned long long *') increases required alignment from 4 to 8 [-Wcast-align]
                         writeVolume((VTK_TT*)(dataPtr), groupId, name.c_str(),
                                     ^~~~~~~~~~~~~~~~~~
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:748:68: note: expanded from macro 'vtkTemplateMacro'
  vtkTemplateMacroCase(VTK_UNSIGNED_LONG_LONG, unsigned long long, call);   \
                                                                   ^
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:743:38: note: expanded from macro 'vtkTemplateMacroCase'
  case typeN: { typedef type VTK_TT; call; }; break
                                     ^
/Users/cory.quammen/src/tomviz/tomviz/EmdFormat.cxx:270:38: warning: cast from 'float *' to 'VTK_TT *' (aka 'long long *') increases required alignment from 4 to 8 [-Wcast-align]
                         writeVolume((VTK_TT*)(dataPtr), groupId, name.c_str(),
                                     ^~~~~~~~~~~~~~~~~~
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:749:48: note: expanded from macro 'vtkTemplateMacro'
  vtkTemplateMacroCase(VTK_ID_TYPE, vtkIdType, call);                       \
                                               ^
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:743:38: note: expanded from macro 'vtkTemplateMacroCase'
  case typeN: { typedef type VTK_TT; call; }; break
                                     ^
/Users/cory.quammen/src/tomviz/tomviz/EmdFormat.cxx:270:38: warning: cast from 'float *' to 'VTK_TT *' (aka 'long *') increases required alignment from 4 to 8 [-Wcast-align]
                         writeVolume((VTK_TT*)(dataPtr), groupId, name.c_str(),
                                     ^~~~~~~~~~~~~~~~~~
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:750:40: note: expanded from macro 'vtkTemplateMacro'
  vtkTemplateMacroCase(VTK_LONG, long, call);                               \
                                       ^
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:743:38: note: expanded from macro 'vtkTemplateMacroCase'
  case typeN: { typedef type VTK_TT; call; }; break
                                     ^
/Users/cory.quammen/src/tomviz/tomviz/EmdFormat.cxx:270:38: warning: cast from 'float *' to 'VTK_TT *' (aka 'unsigned long *') increases required alignment from 4 to 8 [-Wcast-align]
                         writeVolume((VTK_TT*)(dataPtr), groupId, name.c_str(),
                                     ^~~~~~~~~~~~~~~~~~
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:751:58: note: expanded from macro 'vtkTemplateMacro'
  vtkTemplateMacroCase(VTK_UNSIGNED_LONG, unsigned long, call);             \
                                                         ^
/Users/cory.quammen/src/paraview-tomviz/VTK/Common/Core/vtkSetGet.h:743:38: note: expanded from macro 'vtkTemplateMacroCase'
  case typeN: { typedef type VTK_TT; call; }; break
```

--- a/tomviz/EmdFormat.cxx
+++ b/tomviz/EmdFormat.cxx
@@ -234,7 +234,7 @@ public:
     h5dim[2] = dim[0];
 
     auto arrayPtr = data->GetPointData()->GetScalars();
-    auto dataPtr = static_cast<float*>(arrayPtr->GetVoidPointer(0));
+    auto dataPtr = arrayPtr->GetVoidPointer(0);
 
     // Map the VTK types to the HDF5 types for storage and memory. We should
     // probably add more, but I got the important ones for testing in first.

